mantidproject/mantid
# New rotate docs

**Description of work.**
- Added new rotation examples to HowToDefineGeometricShape. Added new plotting examples to MeshPlots. SetSample and CopySample have both been updated already with references to SetGoniometer, but an indentation error in SetSample has been fixed here.
- A section for adding arrows has be added to show how to add the beam arrow and how to add the crystal axes.

**To test:**
- build docs-html target
- check the links and new examples MeshPlotHelp page and HowToDefineGeometricShape page. Check that the plot for each example appears. Note that for some examples the actual plotting script is hidden, and a code-block instead shows the user the important commands.The full script for each plot should be available via `(Source code, png, hires.png, pdf)` above the plot image. Note the MeshObject Plot (which is not a new example) may not work when you build the docs locally as it may not find tube.stl, but it works for the [current online version](https://docs.mantidproject.org/nightly/plotting/MeshPlotHelp.html#id3).
- Also there is a fix in this PR to a example code recently added to SetSample. Check the plot there now appears correctly.

fixes #32031

*This does not require release notes* because **this is documentation**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
